packages/block-editor/src/hooks/color.js

```diff
@@ -61,10 +61,6 @@ const hasLinkColorSupport = ( blockType ) => {
 };
 
 const hasGradientSupport = ( blockType ) => {
-	if ( Platform.OS !== 'web' ) {
```

Comment:
Nice. So this means that mobile is starting to use the hooks?

Reply:
That's correct, we added the color one support and now we are adding it for gradients.